src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/usermodel/User.java

```diff
@@ -0,0 +1,63 @@
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.usermodel;
+
+public class User {
```

Comment:
use lombok, it will replace constructor and getters

Reply:
done